index.html

```diff
@@ -4,134 +4,142 @@
         <meta charset="utf-8">
         <meta name="viewport" content="width=device-width, initial-scale=1">
         <meta http-equiv="X-UA-Compatible" content="IE=edge">
-        
-        <!-- adding required angular JS library and angular JS script-->
-        <script src="js/angular.min.js"></script>
-        <script src="js/gameapp.js"></script>
-        
         <!--Code for jQuery and bootstrap-->
         <script src="js/jquery-3.1.1.min.js"></script>
         <script src="js/bootstrap.min.js"></script>
+        <!-- adding required angular JS library and angular JS script-->
```

Comment:
you should not need to include these files separately, please gulp them to one file